src/InterventionBackend.php

```diff
@@ -76,124 +72,78 @@ class InterventionBackend implements Image_Backend, Flushable
     /**
      * Configure where cached intervention files will be stored
      *
-     * @config
-     * @var string
      */
-    private static $local_temp_path = TEMP_PATH;
+    private static string $local_temp_path = TEMP_PATH;
 
-    /**
-     * @var AssetContainer
-     */
-    private $container;
+    private ?AssetContainer $container = null;
 
-    /**
-     * @var InterventionImage
-     */
-    private $image;
+    private ?InterventionImage $image;
 
-    /**
-     * @var int
-     */
-    private $quality;
+    private int $quality = AbstractEncoder::DEFAULT_QUALITY;
```

Comment:
We need to set this default value because a: we can't pass a null in anymore, and b: there are unit tests that fail if we set it to arbitrary values like 0 or 100.